fyde-access-proxy/install-bm-vm.md

```diff
@@ -22,16 +24,15 @@ nav_order: 1
 1. Download and execute installation script
 
     ```sh
-    sudo bash -c "$(curl -fsSL https://url.fyde.me/install-fyde-proxy-centos)"
+    sudo bash -c "$(curl -fsSL https://url.fyde.me/install-fyde-proxy-linux)"
```

Comment:
@GMartinez-Sisti URL doesn't seem available yet

Reply:
@oNaiPs It will only be available after the merge. But the redirect is already in place:

```sh
→ curl https://url.fyde.me/install-fyde-proxy-linux -I
HTTP/1.1 301 Moved Permanently
Location: https://raw.githubusercontent.com/fyde/docs/master/fyde-access-proxy/scripts/install-fyde-proxy-linux.sh
```